### ISSUE TITLE
Use defensive CSS to renforce accessibility

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#212]
By using stronger selectors like `html[lang]` or `img[alt]`, we could improve developers'  accessibility knowledge:

- by making styles dependent of an accessible markup,
- thus requiring developers to use our HTML templates.

This has many downsides though:
- everything should be explained, at least in the documentation;
- those selectors might only be compiled for non-minified versions (should be equal to dev mode, shouldn't it?)

I guess the best ressource for now is [a11y.css](https://ffoodd.github.io/a11y.css) and its wiki.

FWIW, [Dan Eden implemented an `assert-selector` mixin in Scooter to do just that](https://github.com/dropbox/scooter/pull/4); we may try to include this in non-minified styles only.

Since Boosted assumes its accessibility, I guess it could be done :)

**Comment by MaxLardenois** [comment]
Stale
